Assets/HoloToolkit/SpatialSound/Scripts/AudioOccluder.cs

```diff
@@ -9,47 +9,62 @@ namespace HoloToolkit.Unity
     /// <summary>
     /// Class that implements IAudioInfluencer to provide an occlusion effect.
     /// </summary>
-    public class AudioOccluder : MonoBehaviour, IAudioInfluencer
+    /// <remarks>
+    /// For best results, ensure that all sound emitting objects have an attached
+    /// AudioInfluencerManager. Failing to do so may result in incorrect frequency
+    /// and/or volume restoration when the RemoveEffect method is called.
+    /// </remarks>
+    public class AudioOccluder 
```

Comment:
Do not separate out dependencies on to separate lines as thi makes it harder to read.  The standard used in MRTK is to have all dependencies on the same line as the class.

Reply:
Agreed, there's not many anyway so no need to break them up.